### PR TITLE
Update ampcombi to 0.2.1

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`ampcombi`](https://bioconda.github.io/recipes/ampcombi/README.html): **0.2.0** &rarr; **0.2.1**

[install with bioconda](http://bioconda.github.io/recipes/ampcombi/README.html) [Conda](https://anaconda.org/bioconda/ampcombi/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/ampcombi`](https://github.com//bioconda/bioconda-recipes/tree/bump/ampcombi/recipes/ampcombi) (click to view/edit other files)
Summary | A parsing tool to convert and summarise the outputs from multiple AMP detection tools.
Home | [https://github.com/Darcy220606/AMPcombi](https://github.com/Darcy220606/AMPcombi)
Releases |[https://pypi.org/pypi/AMPcombi/json](https://pypi.org/pypi/AMPcombi/json)
Recipe Maintainer(s) | @darcy220606, @louperelo

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/ampcombi/meta.yaml [hunk 1]
```diff
@@ -22,19 +22,20 @@ requirements:
   host:
     - pandas
     - pip
-    - python >=3.9
+    - python >=3.11
```
Comment (jfy133) [inline recipes/ampcombi/meta.yaml:25]:
```suggestion
    - python ==3.11
```

File: recipes/ampcombi/meta.yaml [hunk 2]
```diff
@@ -22,19 +22,20 @@ requirements:
...
+    - biopython >=1.8
     - backports.tempfile
     - contextlib2
-    - diamond
     - openpyxl
     - jsonschema
-    - python >=3.9
+    - python >=3.11
+    - diamond >=2.0.15
+    - mmseqs2 >=15.6f452
```
Comment (jfy133) [inline recipes/ampcombi/meta.yaml:38]:
```suggestion
    - python ==3.11
    - diamond ==2.0.15
    - mmseqs2 ==15.6f452
```

File: recipes/ampcombi/meta.yaml [hunk 3]
```diff
@@ -22,19 +22,20 @@ requirements:
...
-    - python >=3.9
+    - python >=3.11
   run:
-    - pandas
+    - pandas >=1.5.2
     - requests
     - parsedatetime
     - subprocess32
-    - biopython
+    - biopython >=1.8
```
Comment (jfy133) [inline recipes/ampcombi/meta.yaml:31]:
```suggestion
    - biopython ==1.8
```

File: recipes/ampcombi/meta.yaml [hunk 4]
```diff
@@ -22,19 +22,20 @@ requirements:
   host:
     - pandas
     - pip
-    - python >=3.9
+    - python >=3.11
   run:
-    - pandas
+    - pandas >=1.5.2
```
Comment (jfy133) [inline recipes/ampcombi/meta.yaml:27]:
```suggestion
    - pandas ==1.5.2
```

File: recipes/ampcombi/meta.yaml [hunk 5]
```diff
@@ -22,19 +22,20 @@ requirements:
...
-    - python >=3.9
+    - python ==3.11
   run:
-    - pandas
+    - pandas ==1.5.2
     - requests
     - parsedatetime
     - subprocess32
-    - biopython
+    - biopython ==1.8
```
Comment (jfy133) [inline recipes/ampcombi/meta.yaml:31]:
```suggestion
    - biopython ==1.80
```

**Review by matthdsm — APPROVED** [review]

**Comment by jfy133** [comment]
Added strict pinning at request/requirement of the developer (tool includes internal versionc hecks)

**Pull request merged**